app/controllers/api/articles_controller.rb

```diff
@@ -1,6 +1,7 @@
 class Api::ArticlesController < ApplicationController
   before_action :authenticate_user!, only: %i[create show]
   before_action :is_user_journalist?, only: [:create]
+  before_action :is_user_subscriber?, only: [:show]
```

Comment:
Let's delete this for now, because we might allow registered users 5 free articles!